perma_web/tasks/dev.py

```diff
@@ -701,3 +707,496 @@ def populate_folder_cached_path(ctx, batch_size=500):
             logger.info(f"Stopped with ~ {remaining_to_update} remaining folders to update")
     else:
         logger.info("No more folders left to update!")
+
+#
+# Merge user accounts
+#
+
+TRANSFERRED_ORG_LINKS_CSV = 'merge_reports/tranferred_org_links.csv'
+TRANSFERRED_PERSONAL_LINKS_CSV = 'merge_reports/tranferred_personal_links.csv'
+MERGED_USERS_CSV = 'merge_reports/merged_users.csv'
+RETAINED_USERS_CSV = 'merge_reports/retained_users.csv'
+
+def initialize_csvs():
+    for filename in [TRANSFERRED_ORG_LINKS_CSV, TRANSFERRED_PERSONAL_LINKS_CSV, MERGED_USERS_CSV, RETAINED_USERS_CSV]:
+        os.makedirs(os.path.dirname(filename), exist_ok=True)
+
+    with open(TRANSFERRED_ORG_LINKS_CSV, 'w', newline='') as csvfile:
+        writer = csv.writer(csvfile, delimiter='|')
+        writer.writerow(['guid', 'from user id', 'to user id', 'moved at'])
+
+    with open(TRANSFERRED_PERSONAL_LINKS_CSV, 'w', newline='') as csvfile:
+        writer = csv.writer(csvfile, delimiter='|')
+        writer.writerow(['guid', 'from user id', 'from folder id', 'to user id', 'to folder id', 'moved at'])
+
+    with open(MERGED_USERS_CSV, 'w', newline='') as csvfile:
+        writer = csv.writer(csvfile, delimiter='|')
+        writer.writerow([
+            'user id',
+            'original email',
+            'changed to placeholder',
+            'normalized email',
+            'merged with user id',
+            'org links transferred',
+            'personal links transferred',
+            'merged at'
+        ])
+
+    with open(RETAINED_USERS_CSV, 'w', newline='') as csvfile:
+        writer = csv.writer(csvfile, delimiter='|')
+        writer.writerow(['user id', 'original email', 'normalized email', 'merged with accounts', 'merged at'])
+
+
+def merge_accounts(
+        to_keep,
+        to_delete,
+        copy_memberships=True,
+        transfer_links=False):
+    from perma.models import Link
```

Comment:
Why do we do this with imports in these files? Is it for a reason, or is it a relic of intentions whose shipped has sailed (for instance, a thing for tasks that don't require django to be configured.... or to avoid circular imports in some specific circumstances)?